src/components/TextInputFocusable/index.js

```diff
@@ -189,9 +207,20 @@ class TextInputFocusable extends React.Component {
                         return new File([x], `pasted_image.${extension}`, {});
                     })
                     .then(this.props.onPasteFile)
-                    .catch((error) => {
+                    .catch(() => {
                         const errorDesc = this.props.translate('textInputFocusable.problemGettingImageYouPasted');
-                        alert(`${errorDesc}. \n${error.message}`);
+                        Growl.show(errorDesc, 'error');
+
+                        // We can't paste synthatically as it is blocked from browser due not generated
```

Comment:
Sorry, I don't understand what this comment means... What does "paste synthetically" mean? Can you help me understand why the browser is blocking regular pasting here?

Reply:
So browser only allow paste when it is generated from direct user Action. In Other words, we can allow the original paste event and let browser handle it but we are inside a Promise thus the first condition does not satisfy and paste will not continue.

from MDN https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event
> It's possible to construct and dispatch a synthetic paste event, but this will not affect the document's contents.

Reply:
> So it seems like you implemented a global growl in <Expensify />, but also did not change the usage of the local growl in <ProfilePage />.

I suggested it https://github.com/Expensify/Expensify.cash/pull/3058#issuecomment-846221659

Reply:
Okay, thanks for the explanation. Let's update this comment to make it a bit clearer for future contributors:

```
/* 
 * Since we intercepted the user-triggered paste event to check for attachments,
 * we need to manually set the value and call the `onChangeText` handler.
 * Synthetically-triggered paste events do not affect the document's contents.
 * See https://developer.mozilla.org/en-US/docs/Web/API/Element/paste_event for more details.
 */ 
```